### DISCUSSION TITLE
PackageRule configuration error is generated even though the configuration is correct.

### DISCUSSION THREAD

**Discussion by ddecrulle** [renovatebot/renovate#23167] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I'm trying to add label depend on the file path of the package or lock. 

The document of [matchFileNames](https://docs.renovatebot.com/configuration-options/#matchfilenames) shows some examples.

With this minimal configuration : 

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base"],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchFileNames": ["package.json"],
      "labels": ["container"]
    }
  ]
}
```
An issue with **Action Required: Fix Renovate Configuration** is generated.

It's seems that matchFileNames is not considered as a match selector.

### Relevant debug logs

<details><summary>Logs</summary>

There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

Location: `renovate.json`
Error type: The renovate configuration file contains some invalid settings
Message: `Invalid configuration option: packageRules[0].matchFileNames, packageRules[0]: Each packageRule must contain at least one match* or exclude* selector. Rule: {"matchFileNames":["package.json"],"labels":["container"]}`

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
The hosted app is using v35 still, as v36 was only just release a few hours ago.

https://github.com/renovatebot/renovate/discussions/22738

**Reply by ddecrulle** [reply]
Will version 36 fix this?

**Reply by rarkins** [reply, marked as answer]
Please change `matchFileNames` to `matchFiles` and it will work right away, as well as once v36 is running.

This is because the open source docs reflect v36 which was released today, while the app is running v35
